pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -22,14 +22,15 @@ import (
 )
 
 const (
-	driverBinaryDir      = "/usr/local/bin"
-	kvmDriverBinaryPath  = driverBinaryDir + "/docker-machine-driver-kvm"
-	kvmDriverDownloadURL = "https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.10.0/docker-machine-driver-kvm-centos7"
-	defaultPoolSize      = "20 GB"
-	nodeMac              = "52:fd:fc:07:21:82"
-	nodeIp               = "192.168.126.11"
-	domName              = "crc"
-	poolTemplate         = `<pool type='dir'>
+	driverBinaryDir          = "/usr/local/bin"
+	libvirtDriverCommand     = "crc-driver-libvirt"
+	libvirtDriverBinaryPath  = driverBinaryDir + libvirtDriverCommand
+	libvirtDriverDownloadURL = "https://github.com/code-ready/machine-driver-libvirt/releases/download/0.9.0/crc-driver-libvirt"
+	defaultPoolSize          = "20 GB"
+	nodeMac                  = "52:fd:fc:07:21:82"
+	nodeIp                   = "192.168.126.11"
```

Comment:
This is a tricky one... we can not really assume this, but will leave this for now.